NewsFeed/Common/Views/ScrollableSegmentedControl.swift

```diff
@@ -0,0 +1,235 @@
+//
+//  ScrollableSegmentedControl.swift
+//  ScrollableSegmentedControl
+//
+//  Created by MArko Satlan on 22/07/2018.
+//  Copyright © 2018 MArko Satlan. All rights reserved.
+//
+
+import UIKit
+
+class ScrollableSegmentedControl: UIControl {
+
+// MARK: - Properties
+    // Open
+    open var font = UIFont(name: "Georgia", size: 15) {
+        didSet {
+            setNeedsLayout()
+        }
+    }
+    open var buttonPadding: CGFloat = 15 {
```

Comment:
`buttonSpacing` or `buttonHorizontalSpacing` would be more appropriate name. Padding is more like an "internal" margin, or better said an extra space _inside_ the view.